scalanet/src/io/iohk/scalanet/peergroup/ReqResponseProtocol.scala

```diff
@@ -261,7 +261,11 @@ object ReqResponseProtocol {
     val hostkeyPair = CryptoUtils.genEcKeyPair(rnd, Secp256k1.curveName)
 
     for {
-      config <- Resource.liftF(Task.fromTry(DynamicTLSPeerGroup.Config(address, Secp256k1, hostkeyPair, rnd, None)))
+      config <- Resource.liftF(
+        Task.fromTry(
+          DynamicTLSPeerGroup.Config(address, Secp256k1, hostkeyPair, rnd, useNativeTlsImplementation = false, None)
```

Comment:
becouse `ReqResponseProtocol` is used only in tests, and for time being I do not expect it will be used anywhere else, thats why i am treating it a little bit neglectfully